### PR TITLE
Feature/add inheritance

### PR DESCRIPTION
**Pull request by ANicholasson**

### CODE REVIEW

File: include/tree_option.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+
+#ifndef tree_option_h
+#define tree_option_h
+
+#include <string>
+// Include MenuOption
+#include <menu_option.h>
```
Comment (SundeMarius) [inline include/tree_option.h]: menu_option.h is a local file which is in the same directory. Use #include "menu_option.h" (makes the preprocessor look for it in the same directory as this file).

File: include/menu_option.h [hunk 2]
```diff
@@ -10,23 +10,26 @@
...
-    byte get_number_next_options() const { return numbNextOptions; }
+
+    private:
+        // Member fields
+        // A description of the option (e.g "Feed pet")
+        string description;
+
+    public:
+        // Constructors
+        MenuOption() = default;
```
Comment (SundeMarius) [inline include/menu_option.h]: I don't know why I wrote "= default", you can remove it (compiler knows that MenuOption() is the default constructor by the fact that it takes no input, so not necessary to write "=default").

**Review by SundeMarius — CHANGES_REQUESTED** [review]
To meeget quick fixes

**Review by ANicholasson — COMMENTED** [review]
Fixed in last commit

**Pull request merged**